kickstart/provision.erb

```diff
@@ -207,6 +208,12 @@ fi
 <%= snippet 'saltstack_setup' %>
 <% end -%>
 
+<% if @host.param_true?('disable-uek') -%>
```

Comment:
maybe add `if @host.operatingsystem.name == 'OracleLinux'` here, just to be safe?